Modules/Filtering/ImageGradient/include/itkVectorGradientMagnitudeImageFilter.h

```diff
@@ -225,7 +227,7 @@ class ITK_TEMPLATE_EXPORT VectorGradientMagnitudeImageFilter : public ImageToIma
   }
 #endif
 
-  using WeightsType = FixedArray<TRealType, VectorDimension>;
+  using WeightsType = FixedArray<TRealType, ImageDimension>;
```

Comment:
This change seems fundamental, are you sure of this?
```
  /** Length of the vector pixel type of the input image. */
  static constexpr unsigned int VectorDimension = InputPixelType::Dimension;
```

Reply:
I just realized that this change is what you were discussing with @blowekamp in the associated issue!

Reply:
Pablo, thanks for having had a look. I'm marking as resolved then.